### PR TITLE
docs: update create dashboard section

### PR DESCRIPTION
**Pull request by joaoviana**
Related to: #6370 

### Description:

Updated all screenshots related to dashboards and dashboard filters where we've updated the UI.UX

### CODE REVIEW

**Review by magnew — APPROVED** [review]
Nice work! Looks great. I added a couple of minor language suggestions.

File: docs/docs/get-started/exploring-data/dashboards.md [hunk 1]
```diff
@@ -10,49 +10,59 @@ Dashboards allow you to arrange multiple charts that are related to each other i
...
-Click "Add tile" and then "New chart" to create a new chart. You will be taken to the chart builder. Once you save the chart it will show at the bottom of your
-dashboard.
+Click "Add tile" and then "New chart" to create a new chart. You will be taken to the chart builder. Once you save the chart it will show at the bottom of your dashboard.
 
 Note: These charts can't be used in other dashboards and won't be shown in the global search or in the space.
 
 ![create chart screenshot](./assets/create_chart_in_dashboard.png)
 
-#### How to add existing chart 
+### How to add existing chart
```
[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/get-started/exploring-data/dashboards.md:46]: done!

File: docs/docs/get-started/exploring-data/dashboards.md [hunk 2]
```diff
@@ -10,49 +10,59 @@ Dashboards allow you to arrange multiple charts that are related to each other i
...
 
-Click "Add tile" and then "Saved chart" to see a list of charts that have been saved. Clicking any chart will add it at the bottom of your
-dashboard.
+Click "Add tile" and then "Saved chart" to see a list of charts that have been saved and ready to be added to your dashboard. Clicking any chart will add it at the bottom of your dashboard.
```
[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/get-started/exploring-data/dashboards.md:48]: good shout!

File: docs/docs/get-started/exploring-data/dashboards.md [hunk 3]
```diff
@@ -10,49 +10,59 @@ Dashboards allow you to arrange multiple charts that are related to each other i
...
+You can share your dashboard with anybody in your Lightdash organisation by copying the URL (or by pressing on the 🔗 button). You can also see all of the saved dashboards in your project by clicking on `Browse` --> `All Dashboards`.
 
-![click save chart](./assets/click_save_chart.png)
+![Alt text](all-dashboard-nav.png)
 
-You can share your dashboard with anybody in your Lightdash organisation by copying the URL. You can also see all of the saved dashboards in your project by clicking on `Browse` --> `Dashboards`.
+## 4. Add filters to your dashboards
 
-![browse dashboards](./assets/browse-dashboards.png)
+Follow the guide in [here](/guides/limiting-data-using-filters#adding-filters-to-your-dashboard)
```
[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/get-started/exploring-data/dashboards.md:68]: changed!

File: docs/docs/guides/assets/dashboard-filter-applied.png
[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/guides/assets/dashboard-filter-applied.png:1]: it does not make any sense, you're right. Let me add a better example!

[the remark this answers is not in the document]
Comment (joaoviana) [reply docs/docs/guides/assets/dashboard-filter-applied.png:1]: done and replaced!

**Pull request merged**